homeassistant/components/lcn/__init__.py

```diff
@@ -0,0 +1,118 @@
+"""
+Connects to LCN platform.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/lcn/
+"""
+
+import logging
+import voluptuous as vol
+
+from homeassistant.const import (
+    CONF_HOST, CONF_NAME, CONF_PASSWORD, CONF_PORT, CONF_USERNAME)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+
+from .core import (
+    CONF_ADDRESS, CONF_CONNECTIONS, CONF_DIM_MODE, CONF_SK_NUM_TRIES,
+    DIM_MODES, get_connection)
+
+REQUIREMENTS = ['pypck==0.5.5']
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'lcn'
+DATA_LCN = 'lcn'
+LIB_LCN = 'pypck'
+DEFAULT_NAME = 'pchk'
+
+CONNECTION_SCHEMA = vol.Schema({
+    vol.Required(CONF_HOST): cv.string,
+    vol.Required(CONF_PORT): cv.positive_int,
```

Comment:
We have `cv.port` to check if it's a valid port number.

Reply:
Replaced.